packages/blocks/src/store/reducer.js

```diff
@@ -43,6 +43,47 @@ export function blockTypes( state = {}, action ) {
 	return state;
 }
 
+/**
```

Comment:
I think we also need to change blockTypes to remove styles from the blockTypes, right now they are duplicated in blockTypes and blockStyes.

Reply:
I thought about it. I actually even had it implemented, but decided to revert. The idea is that, this could serve as a way to differentiate which styles are custom and which are defined in the block settings. But I don't have strong opinions though, I'm fine omitting them.